xbmc/application/Application.cpp

```diff
@@ -2450,9 +2450,17 @@ bool CApplication::PlayFile(CFileItem item, const std::string& player, bool bRes
   if (!(options.startpercent > 0.0 || options.starttime > 0.0) &&
       (item.IsBDFile() || item.IsDiscImage()))
   {
-    //check if we must show the simplified bd menu
-    if (!CGUIDialogSimpleMenu::ShowPlaySelection(item))
-      return true;
+    // No video selection when using an external player, because it needs to handle that on its own.
+    const std::string defaulPlayer{
+        player.empty() ? m_ServiceManager->GetPlayerCoreFactory().GetDefaultPlayer(item) : player};
+    const bool isExternalPlayer{
+        m_ServiceManager->GetPlayerCoreFactory().GetPlayerType(defaulPlayer) == "external"};
+    if (!isExternalPlayer)
+    {
+      //check if we must show the simplified bd menu
```

Comment:
Please add a space after the comment marks

Reply:
Just indented an existing code block, the comment was there, but sure.

Reply:
done.